scripts/helmcharts/openreplay/templates/job.yaml

```diff
@@ -141,6 +146,10 @@ spec:
             value: "{{ .Values.fromVersion }}"
           - name: CHART_APP_VERSION
             value: "{{ .Chart.AppVersion }}"
+          - name: CLICKHOUSE_HOST
+            value: "{{ .Values.clickhouse.global.clickhouse.clickhouseHost }}"
+          - name: CLICKHOUSE_PORT
+            value: "{{ .Values.clickhouse.global.clickhouse.clickhousePort }}"
```

Comment:
Have to update these hosts.

Reply:
this should be updated with the latest commits